### PR TITLE
Add Participant Selector to BSDT Wizard

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Any net new LWC work has Sa11y tests & 50% or above lines JEST test coverage~~
~~CRUD/FLS is enforced in Apex~~
~~Permission sets are updated to account for CRUD|FLS|Tab|Classes~~
~~Field sets are updated to account for new fields~~
- [ ] UX approval or UX not necessary
- [ ] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: W-0000000: Work Name (https://github.com/SalesforceFoundation/PMM/pull/303)
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
~~PR contains draft release notes~~
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-8922120](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000097DgGIAU/view)

**Comment by jjbennett** [comment]
Created from the wrong branch. Recreating momentarily.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -66,25 +66,32 @@
...
+        <shortDescription>Title for the Bulk Service Delivery Tool Group Option</shortDescription>
         <value>Create Service Deliveries for a Group of Contacts</value>
     </labels>
     <labels>
         <fullName>BSDT_Group_Option_Description</fullName>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Description for the Bulk Service Delivery Tool Group Data Entry Option</shortDescription>
+        <shortDescription>Description for the Bulk Service Delivery Tool Group Option</shortDescription>
         <value>Enter service deliveries for a group of contacts. For example a workshop or a class.</value>
```
Comment (jessicahaas1) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:88]: <value>Enter Service Delivery values for a group of Contacts. For example, a workshop or a class.</value>

**Review by jessicahaas1 — COMMENTED** [review]
Looks great

**Pull request closed without merging**